R/AddCardModule.R

```diff
@@ -4,18 +4,28 @@
 #'
 #' For more details see the vignette: `vignette("simpleReporter", "teal.reporter")`.
 #' @param id `character(1)` this `shiny` module's id.
+#' @param label `character(1)` label before icon, if used then dynamic hover label is not available.
+#' By default `NULL` so a dynamic hover label is used.
 #' @return `shiny::tagList`
 #' @export
-add_card_button_ui <- function(id) {
+add_card_button_ui <- function(id, label = NULL) {
```

Comment:
no need for `label`, this will be standard.

Reply:
done